src/java.desktop/share/classes/java/awt/HeadlessException.java

```diff
@@ -48,7 +48,9 @@ public class HeadlessException extends UnsupportedOperationException {
     private static final long serialVersionUID = 167183644944358563L;
 
     /**
-     * Constructs new {@code HeadlessException} with empty message.
+     * Constructs a new {@code HeadlessException} with {@code null} as its detail message.
+     * The default headless message may replace {@code null} in some cases, as outlined below.
+     * <p>
      * For such {@code HeadlessException} the default headless error message
      * may be auto-generated for some platforms.
      * The text of the default headless message may depend on
```

Comment:
This doesn't align, the following text has to be modified too.

The first sentence of the second paragraph, “For such `{@code HeadlessException}` the default headless error message may be auto-generated for some platforms,” should be replaced with the slightly edited text from the `HeadlessException(String)` constructor: “For some platforms, the `{@code null}` detail message may be replaced with the default headless error message.”

Reply:
I don't see why.